### DISCUSSION TITLE
New albums aren't based on my artists

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1186] [General]
The quick picks settings mentioned new albums are based on artists i listen to but it looks like the app is fetching new albums based on country and not based on my interest. Is it possible to fix it or change the location manually for better results?

![IMG_20240323_124732](https://github.com/fast4x/RiMusic/assets/99175083/856264a8-4dd5-4cb4-9712-b498f0986c57)

**Comment by fast4x** [comment]
I can try to change geographic location, please create a ticket. 😊
